app/registries/foreman/plugin.rb

```diff
@@ -77,11 +77,6 @@ def global_registry(name, registry)
         end
       end
 
-      def medium_providers
```

Comment:
Looks like katello is still using this one...

Reply:
https://github.com/Katello/katello/pull/9813

Reply:
looks like this deprecation was added to the wrong place, the instance method wasn't deprecated and replaced with the new name. Updated it, but we'll need to either merge the katello pr before this (and break katello) to get katello tests green here or merge this pr (and break katello) before getting tests green on the katello PR. or merge both at the same time and hope nothing else breaks.

Reply:
Shall we then have a separate PR just for a better visibility?

Isn't the instance method just calling the class method?

Reply:
🤔 that could make sense and allow merging the katello fix before this one, https://projects.theforeman.org/issues/34002